covid_xprize/standard_predictor/predict.py

```diff
@@ -7,11 +7,6 @@
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 
-# Fixed weights for the standard predictor.
-MODEL_WEIGHTS_FILE = os.path.join(ROOT_DIR, "models", "trained_model_weights.h5")
-
-DATA_FILE = os.path.join(ROOT_DIR, 'data', "OxCGRT_latest.csv")
```

Comment:
The 2 constants are moved to `xprize_predictor.py`. `DATA_FILE` was already there actually.